package.json

```diff
@@ -14,11 +14,13 @@
   ],
   "type": "module",
   "files": [
-    "index.js"
+    "index.js",
+    "launchSearch.js"
   ],
   "scripts": {
     "prepare:husky": "husky install",
     "prepare:esbuild": "esbuild index.ts --bundle --packages=external --outfile=index.js --platform=node --format=esm --tree-shaking=true",
+    "prepare:docker": "esbuild launchSearch.ts --bundle --packages=external --outfile=launchSearch.js --platform=node --format=esm --tree-shaking=true",
```

Comment:
Please combine these into a single esbuild command that bundles multiple entry points. You might find some inspiration in https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/esbuild.js.